geoarrow-types/src/geoarrow/types/crs.py

```diff
@@ -0,0 +1,182 @@
+from copy import deepcopy
+import json
+from typing import Union, Mapping
+
+
+class Crs:
```

Comment:
I'd tend to suggest a Protocol here, because the `pyproj.CRS` doesn't need to subclass from this `Crs` object.